packages/components/src/sandbox/index.native.js

```diff
@@ -142,7 +181,7 @@ function Sandbox( {
 		// Scripts go into the body rather than the head, to support embedded content such as Instagram
 		// that expect the scripts to be part of the body.
 		const htmlDoc = (
-			<html lang={ lang } className={ type }>
+			<html lang={ lang }>
```

Comment:
I realized that adding `type` to the `html` tag breaks the styles for the `
Resize for smaller devices` option, besides, I think this is not really necessary as we're already adding it in the `body` tag.